src/main/resources/hudson/tasks/junit/History/index.jelly

```diff
@@ -75,5 +73,7 @@ THE SOFTWARE.
 	      </j:if>	       
       	</div>
     </l:main-panel>
-	</l:layout>
+	</bs:layout>
```

Comment:
I have the scripts within layout

Reply:
I'll push that but if I have it there i get:

![image](https://user-images.githubusercontent.com/21194782/93095479-0d328580-f69b-11ea-9329-dda4daae3628.png)

Which I think it caused by the JavaScript running before the DOM is present:

```
    const chartPlaceHolder = document.getElementById(chartDivId);
    const chart = echarts.init(chartPlaceHolder);
```

`chartPlaceHolder` is undefined from zoomable-trend-chart.js

Reply:
Does the card show up? It is placed in an `if`?

Reply:
card shows up yes.

![image](https://user-images.githubusercontent.com/21194782/93101903-c2b50700-f6a2-11ea-9be1-07202fc52f13.png)

Reply:
hmm it seems to run multiple times and sometimes it's there and sometimes it's now =/